docker-compose.yml

```
@@ -25,6 +25,20 @@ services:
         source: postgres_data_test
         target: /var/lib/postgresql/data
 
+  mailpit-dev:
+    extends:
+      file: common-services.yml
+      service: mailpit
+    container_name: sgn_mailpit_dev
+    ports:
+      - 8025:8025
+      - 1025:1025
+    volumes:
+      - type: volume
+        source: mailpit_data_dev
+        target: /data
+
```

```suggestion

  mailpit-test:
    extends:
      file: common-services.yml
      service: mailpit
    container_name: sgn_mailpit_test
    ports:
      - 8026:8025
      - 1026:1025
    volumes:
      - type: volume
        source: mailpit_data_test
        target: /data
```
+ le nouveau volume pour simplifier les tests

---

Il me semblait qu'on était tombé d'accord pour ne pas changer cette configuration dans le cadre de cette PR, vu que ça ne change rien à la _feature_. Quelque chose a changé ?